platforms/chibios/drivers/backlight_pwm.c

```diff
@@ -28,9 +28,9 @@
 // Support for pins which are on TIM1_CH1N
 #ifdef BACKLIGHT_PWM_COMPLEMENTARY_OUTPUT
 #    if BACKLIGHT_ON_STATE == 1
```

Comment:
Complementary output is inverted, so this configuration was intentional and should not be changed.

Reply:
> Complementary output is inverted, so this configuration was intentional. Has this actually been validated to be not the case?
Yes, just like sigprof explain. The inverted wasn't be the case, unless both the normal and complementary are in use.

> These backlight and rgb light bug fixes should also be submitted separately.
Thanks, I'll try to split it out.